index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Work+Sans:wght@500&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="aboutme">
+      <h1>Andrea Carlota</h1>
+      <p>
+        I’m a Graphic Designer from Caracas, Venezuela interested in design,
+        fashion, illustration, architecture and getting the work done.
+      </p>
+      <ul class="contact-info">
+        <li class="contact-info">agonzaleznoriega@mica.edu</li>
+        <li class="contact-info">andreacarlotagn@gmail.com</li>
+        <li class="contact-info">240-3169812</li>
+      </ul>
+    </header>
+
+    <main>
+      <section class="experience" id="experience">
+        <h2>Experience</h2>
+
+        <h3>Amazon - Warehouse Associate</h3>
+        <div>
+          <p>June 2019 - Present | Baltimore, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Fenton's Uniforms - Shipping Associate</h3>
+        <div>
+          <p>May 2018 - August 2019 | Silver Spring, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Cafritz Art Store - Key Holder</h3>
+        <div>
+          <p>October 2016 - February 2018 | Takoma Park, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Excalibur Student Newspaper - Layout Designer</h3>
+        <div>
+          <p>May 2016 - May 2018. | Silver Spring, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Excalibur Student Newspaper - Chief Editor</h3>
+        <div>
+          <p>May 2017 - May 2018. | Silver Spring, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+      </section>
+
+      <section id="education">
+        <h2>Education</h2>
+
+        <h3>Maryland Institute of College of Art</h3>
+        <div>
+          
+        <p>Baltimore, MD. 2019-Present. BFA in Graphic Design.</p></div>
+
+        <h3>Montgomery College</h3>
+       <div> <p>
+          Silver Spring, MD. 2016 - 2019. AFA in Graphic Design. Student
+          Leadership Recognition, 2017, 2018. Dean’s List 2016 - 2017.
+         </p></div>
+
+        <h3>Central University of Venezuela.</h3>
+       <div> <p>Caracas, Venezuela. 2012-2016. Incomplete. Architecture.</p>
+        </div> </section>
+
+      <--! cant make this sections one next to the other... cant make the h4 next to the description... ugh--> 
```

Comment:
Your HTML comment is showing up in the actual web page because the exclamation point is in the wrong spot:
```suggestion
      <!-- cant make this sections one next to the other... cant make the h4 next to the description... ugh --> 
```

Reply:
I'll address this comment when I get into your CSS file 👍🏻